packages/stateless/src/lib/provider/provider.interface.ts

```diff
@@ -56,6 +56,7 @@ export interface IPushOptions {
     channelId?: string;
     categoryId?: string;
     mutableContent?: boolean;
+    fcmOptions?: { [key: string]: string };
```

Comment:
You are duplicating the property in the interface (line 61) and removing its functionality. When merging, please don't remove `{ analyticsLabel?: string };` and include the generic record.

Reply:
Hi @p-fernandez 
I clean up the PR, removed duplicated changes